homeassistant/components/dlib_face_identify/image_processing.py

```diff
@@ -17,6 +17,7 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_FACES): {cv.string: cv.isfile},
+    vol.Optional(CONF_CONFIDENCE, default=0.6): cv.string,
```

Comment:
Why is this validated with `cv.string`? Shouldn't it validate as a float (`vol.Coerce(float)`)?

Reply:
Yes, CONF_CONFIDENCE is a float, changes committed, thanks!